src/test/groovy/org/gradle/profiler/ChromeTraceIntegrationTest.groovy

```diff
@@ -17,7 +17,7 @@ class ChromeTraceIntegrationTest extends AbstractProfilerIntegrationTest {
         new File(outputDir, "${versionUnderTest}-trace.html").isFile()
 
         where:
-        versionUnderTest << supportedGradleVersions
+        versionUnderTest << ["4.7"]
```

Comment:
Why not test with all versions as before?

Reply:
Good catch, I changed this when testing out something locally. Reverted to testing all versions again.